src/main/java/org/verdictdb/jdbc41/VerdictResultSetMetaData.java

```diff
@@ -36,6 +36,11 @@ public VerdictResultSetMetaData(VerdictSingleResult queryResult) {
     this.queryResult = queryResult;
   }
 
+  public boolean isAggregate(int column) {
```

Comment:
Why is this method needed? This does not seem to be a standard JDBC method.

Reply:
I see. It seems I never use this method. I will remove it.